transmit-lib/src/cmr/transmit/urs.clj

```diff
@@ -59,6 +63,25 @@
        response
        (not (nil? response))))))
 
+(defn get-user-info
+  "Returns URS info associated with a username"
+  [context user]
+  (let [{:keys [status body]} (request-with-auth context {:url-fn #(user-info-url % user)
+                                                          :method :get
+                                                          :raw? true})]
+   (when-not (= 200 status)
+     (errors/internal-error!
+      (format "Cannot get info for username [%s] in URS. Failed with status code [%d].
+               EDL error message: [%s]"
+              user status (str body))))
```

Comment:
use `(pr-str body)`.

Reply:
Did not know about this function - now using it.